testkit/src/main/scala/org/bitcoins/testkit/chain/ChainUnitTest.scala

```diff
@@ -436,15 +436,13 @@ object ChainUnitTest extends ChainVerificationLogger {
       appConfig: ChainAppConfig): Future[(ChainHandler, BlockHeaderDb)] = {
     val tableSetupF = setupAllTables()
 
-    val chainHandlerF = makeChainHandler()
-
     val genesisHeaderF = tableSetupF.flatMap { _ =>
+      val chainHandlerF = makeChainHandler()
       for {
         chainHandler <- chainHandlerF
```

Comment:
super nit: Any reason not to simply
```
for {
    chainHandler <- makeChainHandler()
...
```

Reply:
no, but i'm not going to do it because I don't want to wait for the build :-)